rotkehlchen/api/v1/schemas.py

```diff
@@ -1581,11 +1586,29 @@ class ExchangesResourceAddSchema(Schema):
     name = fields.String(required=True)
     location = LocationField(limit_to=SUPPORTED_EXCHANGES, required=True)
     api_key = ApiKeyField(required=True)
-    api_secret = ApiSecretField(required=True)
+    api_secret = ApiSecretField(load_default=b'')
     passphrase = fields.String(load_default=None)
     kraken_account_type = SerializableEnumField(enum_class=KrakenAccountType, load_default=None)
     binance_markets = fields.List(fields.String(), load_default=None)
 
+    @validates_schema
+    def validate_schema(
+            self,
+            data: dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        if (location := data['location']) not in EXCHANGES_WITHOUT_API_SECRET and not data.get('api_secret'):  # noqa: E501
+            raise ValidationError(
+                f'{location.name.title()} requires an API secret',
+                field_name='api_secret',
+            )
```

Comment:
check first the condition of not data.get('api_secret') since most of the exchanges have them and avoid one extra check